torchvision/prototype/features/_bounding_box.py

```diff
@@ -66,15 +66,23 @@ def to_format(self, format: Union[str, BoundingBoxFormat]) -> BoundingBox:
             format = BoundingBoxFormat.from_str(format.upper())
 
         return BoundingBox.wrap_like(
-            self, self._F.convert_format_bounding_box(self, old_format=self.format, new_format=format), format=format
+            self,
+            self._F.convert_format_bounding_box(
+                self.as_subclass(torch.Tensor).as_subclass(torch.Tensor), old_format=self.format, new_format=format
```

Comment:
Why `as_subclass(torch.Tensor).as_subclass(torch.Tensor)`? Did you hit replace twice?

Reply:
Most likely when I was fixing https://github.com/pytorch/vision/pull/6807#discussion_r1001158327. Good catch though!